CONTRIBUTING.md

```diff
@@ -1,12 +1,9 @@
 # Contributing to opentelemetry-python
 
-The Python special interest group (SIG) meets regularly. See the OpenTelemetry
-[community](https://github.com/open-telemetry/community#python-sdk) repo for
-information on this and other language SIGs.
+The Python special interest group (SIG) meets weekly on Thursdays at 9AM PST. Check the [OpenTelemetry community calendar](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com) for specific dates and Zoom meeting links.
 
 See the [public meeting notes](https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs/edit)
-for a summary description of past meetings. To request edit access, join the
```

Comment:
Curious why are we removing the part about requesting edit access?

Reply:
Anyone can edit this document, and they don't even have to sign in. I don't know when this changed or has it always been like this?

Reply:
Ahh interesting. Not sure if this has always been the case.